npm/publish-wiki.js

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env node
+// ----------------------------------------------------------------------------------------------------------------------
+// This script is intended to publish wiki of this module
+// ----------------------------------------------------------------------------------------------------------------------
+
+require('shelljs/global');
+require('colors');
+
+// Stop on first error
+set('-e');
+console.log('Publishing wiki...'.yellow.bold);
+
+var publishResult,
+
+    WIKI_URL = 'https://github.com/postmanlabs/postman-collection.wiki.git',
+    WIKI_GIT_PATH = '.tmp/github-wiki',
+    WIKI_VERSION = exec('git describe --always').stdout;
+
+// build the reference MD
+exec('npm run build-wiki');
+
+// clone repository
+mkdir('-p', WIKI_GIT_PATH);
+rm('-rf', WIKI_GIT_PATH);
+exec(`git clone ${WIKI_URL} ${WIKI_GIT_PATH} --quiet`);
+
+// update contents of repository
+exec('node ./wiki_create_content_from_reference.js');
+
+// silence terminal output to prevent leaking sensitive information
+config.silent = true;
+
+pushd(WIKI_GIT_PATH);
+exec('git add --all');
+exec(`git commit -m "[auto] ${WIKI_VERSION}"`);
+publishResult = exec('git push origin master');
+
+if (publishResult.code !== 0) {
```

Comment:
It would be possible to mimic `trap` based behavior by using `process.on('SIGINT', ...)`